components/widgets/inputs/emails_input.jsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import Creatable from 'react-select/lib/Creatable';
+import {components} from 'react-select';
+import {FormattedMessage} from 'react-intl';
+
+import {isEmail} from 'mattermost-redux/utils/helpers';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import MailIcon from 'components/svg/mail_icon';
+import MailPlusIcon from 'components/svg/mail_plus_icon';
+
+import './emails_input.scss';
+
+export default class EmailsInput extends React.Component {
+    static propTypes = {
+        placeholder: PropTypes.string,
+        onChange: PropTypes.func,
+        value: PropTypes.arrayOf(PropTypes.string),
+    }
+
+    getCreateLabel = (value) => (
+        <React.Fragment>
+            <MailPlusIcon className='mail-plus-icon'/>
+            <FormattedMarkdownMessage
+                key='widgets.emails_input.valid_email'
+                id='widgets.emails_input.valid_email'
+                defaultMessage='Invite **{email}** as a channel guest'
```

Comment:
This text seems very specific for a component called `EmailsInput`.

Reply:
Fixed